AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -225,7 +225,7 @@ - (void)sendFirstSessionUpdateLog {
   MSDistributionStartSessionLog *log = [[MSDistributionStartSessionLog alloc] init];
 
   // Send log to log manager.
-  [self.channelUnit enqueueItem:log];
+  [self.channelUnit enqueueItem:log flags:MSFlagsNone];
```

Comment:
Should it be `Normal`?

Reply:
None needs to map to default values for each flags as discussed.